eventnotifier/slack/slack.go

```diff
@@ -228,28 +228,59 @@ func (n *Notifier) buildSlackMessage(obj client.Object, event corev1.Event, noti
 	infoBlocks := n.buildSlackBlocks(obj, notifType, reporting)
 
 	var (
-		userInfo authv1.UserInfo
-		err      error
+		userEmail string
+		userInfo  authv1.UserInfo
+		err       error
 	)
 
+	//two options: make this entire thing apart of a condition so if email doesn't exist then do this current
+	//otherwise if email doees exist, some code logic and rather than user info I return slack message with just email
+	//however its entirely possible I can't do that so I have to look at what slack message is expecting
+
 	switch d := obj.(type) {
 	case *v1beta1.Disruption:
+		if d.Spec.Reporting.SlackUserEmail != "" {
+			userEmail = d.Spec.Reporting.SlackUserEmail
+		}
+
 		userInfo, err = d.UserInfo()
 	case *v1beta1.DisruptionCron:
+		if d.Spec.Reporting.SlackUserEmail != "" {
+			userEmail = d.Spec.Reporting.SlackUserEmail
+		}
 		userInfo, err = d.UserInfo()
 	}
 
 	if err != nil {
 		logger.Warnw("unable to retrieve user info", "error", err)
 	}
 
+	if userEmail == "" {
+		userEmailAddress, err := mail.ParseAddress(userInfo.Username)
+
+		userEmail = userEmailAddress.Address
+
+		if err != nil {
+			logger.Warnw("unable to retrieve user info", "error", err)
+		}
+	}
+
+	//do the parse regardless to make sure we have a valid email
+	userEmailAddress, err := mail.ParseAddress(userEmail)
+
+	userEmail = userEmailAddress.Address
+
+	if err != nil {
+		logger.Warnw("unable to retrieve user info", "error", err)
+	}
```

Comment:
I think we might be able to replace all of L258-L275 with:
```suggestion
if userEmail == "" {
  userEmail = userInfo.Username
}
```

wdyt? We'll keep the same email correctness check in `sendMessageToUserChannel`. That will retain the old behavior